packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list/page-menu.tsx

```diff
@@ -56,6 +58,21 @@ export const PageMenu: VFC<PageMenuProps> = ({ popupState, entityId }) => {
           }, 2000);
         },
       },
+      {
+        title: "Archive page",
+        icon: faArchive,
+        onClick: async () => {
+          try {
+            // @todo handle loading/error states properly
+            await archivePage(entityId);
+          } catch (err) {
+            // eslint-disable-next-line no-console -- TODO: consider using logger
+            console.log("err ==> ", err);
```

Comment:
Makes it slightly easier to see what has gone wrong, if it does
```suggestion
            console.log("Error archiving page: ", err);
```